encryption/three-bears/ccakem-cshake256/impl/skylake.yaml

```diff
@@ -0,0 +1,8 @@
+name: Skylake
+platform: c, assembly
+type: optimized
+sources:
+  - 'Implementation code (https://sourceforge.net/projects/threebears/)'
+code size:
```

Comment:
I'd add dependencies here, and also add in a comment that those are not included in the code size. What do you think? (applies to all impl files)

From the specs:

> The code size does not include Keccak, since we linked an
external Keccak library